cep-9999.md

```diff
@@ -0,0 +1,78 @@
+# CEP ???? - Names in conda packages and channels
+
+<table>
+<tr><td> Title </td><td> CEP ???? - Names in conda packages and channels </td>
+<tr><td> Status </td><td> Draft </td></tr>
+<tr><td> Author(s) </td><td> Jaime Rodríguez-Guerra &lt;jaime.rogue@gmail.com&gt; <br /> Matthew R. Becker &lt;becker.mr@gmail.com&gt;</td></tr>
+<tr><td> Created </td><td> Mar 11, 2025</td></tr>
+<tr><td> Updated </td><td> Mar 11, 2025</td></tr>
+<tr><td> Discussion </td><td> N/A </td></tr>
+<tr><td> Implementation </td><td> N/A </td></tr>
+</table>
+
+## Abstract
+
+This CEP aims to standardize names and other strings used throughout the conda ecosystem.
+
+## Specification
+
+### Artifacts
+
+The conda ecosystem distinguishes between two types of packages:
+
+- Regular package names: represented by a concrete, downloadable, extractable conda artifact.
```

Comment:
I see now where "regular" comes from. Sorry I missed that. Maybe we can pick another term? Unfortunately I don't have any great suggestions. :/

Reply:
Yea this is one of the areas where I want a CEP too. Standardized terminology for each thing. Package, artifact, distribution... is too loaded. Let's go for distributable package for now!